sdk/storage/azblob/blob/constants.go

```diff
@@ -171,12 +172,24 @@ func PossibleDeleteTypeValues() []DeleteType {
 // ExpiryOptions defines values for ExpiryOptions
 type ExpiryOptions = generated.ExpiryOptions
 
-const (
-	ExpiryOptionsAbsolute           ExpiryOptions = generated.ExpiryOptionsAbsolute
-	ExpiryOptionsNeverExpire        ExpiryOptions = generated.ExpiryOptionsNeverExpire
-	ExpiryOptionsRelativeToCreation ExpiryOptions = generated.ExpiryOptionsRelativeToCreation
-	ExpiryOptionsRelativeToNow      ExpiryOptions = generated.ExpiryOptionsRelativeToNow
-)
+// ExpiryType defines values for ExpiryType
+type ExpiryType interface {
+	format(o *SetExpiryOptions) (generated.ExpiryOptions, *generated.BlobClientSetExpiryOptions)
+}
+
+// ExpiryTypeAbsolute defines the absolute time for the blob expiry
+type ExpiryTypeAbsolute time.Time
+
+// ExpiryTypeRelativeToNow defines the duration relative to now for the blob expiry
+type ExpiryTypeRelativeToNow time.Duration
+
+// ExpiryTypeRelativeToCreation defines the duration relative to creation for the blob expiry
+type ExpiryTypeRelativeToCreation time.Duration
+
+// ExpiryTypeNeverExpire defines that the blob will be set to never expire
+type ExpiryTypeNeverExpire struct {
```

Comment:
You can remove the "Expire" suffix as it already starts with the word "Expiry".

Reply:
Done